crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -295,9 +295,13 @@ bitflags::bitflags! {
         const OCCLUSION_TEXTURE          = (1 << 3);
         const DOUBLE_SIDED               = (1 << 4);
         const UNLIT                      = (1 << 5);
-        const ALPHA_MODE_OPAQUE          = (1 << 6);
-        const ALPHA_MODE_MASK            = (1 << 7);
-        const ALPHA_MODE_BLEND           = (1 << 8);
+        const ALPHA_MODE_BITS            = (0b111 << 6);
```

Comment:
Can we have a small comment explaining what this bitflag is for / how the bitflags for the alpha modes are arranged?

Reply:
I added a comment, and changed it to keep the binary literal just for the mask, using sequential decimals for the values to show there's no special bitwise magic going on (except for the mask). Also renamed the masks for consistency with other reserved bits masks we already had.